simulators/gaia/__init__.py

```diff
@@ -151,104 +157,110 @@ def _execute(self, msg):
             if args[1] not in range(1024):
                 return self._error(1010)
         cmd = getattr(self, cmd)
-        return cmd(args)
+        raw_response = cmd(args)
+        tail = ' %s%s' % (self.cmd_id, self.tail)
+        return self.header + str(raw_response) + tail
```

Comment:
Not sure the header should be sent back in the answer, in the original report it was not present, but maybe it got changed